apps/web/next.config.js

```diff
@@ -262,6 +262,30 @@ const nextConfig = {
 
     return redirects;
   },
+  async headers() {
+    const ContentSecurityPolicy = `
+      default-src 'self';
```

Comment:
CSP policy can't be simply implemented like this. There are many errors that need to be sorted out first
See https://capture.dropbox.com/qu4TOd4jXCNqacK5

Example errors are
- We using inline scripts which is not allowed because `unsafe-inline` is not allowed in CSP.
- There seems to be use of some data URIs which needs to be allowed.